tools/ColorTool/ColorTool/Program.cs

```diff
@@ -200,6 +310,41 @@ static bool SetProperties(uint[] colorTable)
             }
             return success;
         }
+
+        static bool SetPropertiesWithVt(uint[] colorTable)
+        {
+            int STD_OUTPUT_HANDLE = -11;
```

Comment:
Err... if this isn't defined somewhere already in C# land, can you please keep this in the ConsoleApi.cs file with all the other definitions brought in from C/C++ header files?